components/org.wso2.carbon.deployment.notifier/src/test/java/org/wso2/carbon/deployment/notifier/deployers/CustomDeployer.java

```diff
@@ -80,6 +90,9 @@ public String deploy(Artifact artifact) throws CarbonDeploymentException {
             if (content.contains("sample1")) {
                 sample1Deployed = true;
                 key = artifact.getName();
+            } else if (content.contains("sample2")) {
```

Comment:
Shall we turn-around the equal check for avoiding any NPEs ?
For ex: "something".equals(string)